packages/std/src/results/cosmos_msg.rs

```diff
@@ -59,6 +59,32 @@ pub enum CosmosMsg<T = Empty> {
     Gov(GovMsg),
 }
 
+impl<T> CosmosMsg<T> {
+    /// Convert this this [`CosmosMsg<T>`] to a [`CosmosMsg<U>`] with a different custom message type.
+    /// This allows easier interactions between code written for a specific chain and
+    /// code written for multiple chains.
+    /// If this is the [`CosmosMsg::Custom`] variant, the function returns `None`.
+    pub fn change_custom<U>(self) -> Option<CosmosMsg<U>> {
+        Some(match self {
+            CosmosMsg::Bank(msg) => CosmosMsg::Bank(msg),
+            CosmosMsg::Custom(_) => return None,
+            #[cfg(feature = "staking")]
+            CosmosMsg::Staking(msg) => CosmosMsg::Staking(msg),
+            #[cfg(feature = "staking")]
+            CosmosMsg::Distribution(msg) => CosmosMsg::Distribution(msg),
+            #[cfg(feature = "stargate")]
+            CosmosMsg::Stargate { type_url, value } => CosmosMsg::Stargate { type_url, value },
+            #[cfg(feature = "cosmwasm_2_0")]
+            CosmosMsg::Any(msg) => CosmosMsg::Any(msg),
+            #[cfg(feature = "stargate")]
+            CosmosMsg::Ibc(msg) => CosmosMsg::Ibc(msg),
+            CosmosMsg::Wasm(msg) => CosmosMsg::Wasm(msg),
+            #[cfg(feature = "stargate")]
+            CosmosMsg::Gov(msg) => CosmosMsg::Gov(msg),
+        })
```

Comment:
Oh sweet, no default needed here such that return `None` is only possible in one case